### DISCUSSION TITLE
item.yml custom commands

### DISCUSSION THREAD

**Discussion by HierarchyMan** [diamond-rip/Eden#116] [Ideas]
@GoodestEnglish remember the time when I asked you where the option for duel arena selection was
turns out, my core also had /settings command and every time I right clicked Eden settings clock it would open up core settings menu and not eden settings menu
is there a way to fix this?
maybe you could allow custom commands?

**Comment by GoodestEnglish** [comment]
It will be fixed in next update

**Comment by HierarchyMan** [comment]
also add clone kit command so its easier to set up similar kits quickly
